.github/workflows/build.yml

```
@@ -35,7 +35,7 @@ jobs:
           token: ${{ secrets.CODECOV_TOKEN }}
           file: ./coverage.txt
       - uses: goreleaser/goreleaser-action@v6
-        if: success() && startsWith(github.ref, 'refs/tags/') && matrix.os == 'ubuntu-latest' && matrix.go-version == 'stable'
+        if: success() && startsWith(github.ref, 'refs/tags/') && matrix.os == 'ubuntu-latest' && matrix.go-version == 'stable' # editorconfig-checker-disable-line
```

I think it could be added on the line before no?

It could help to add a comment about the reason

```suggestion
        # editorconfig-checker-disable-line : the line length is a best effort here there is nothing we can do unfortunately
        if: success() && startsWith(github.ref, 'refs/tags/') && matrix.os == 'ubuntu-latest' && matrix.go-version == 'stable'
```

---

Please note this comment would be the sane for every inline comment you added to disable the linter

---

Apparently it's not possible 🤔😒

- https://github.com/editorconfig-checker/editorconfig-checker/issues/180

But the idea of adding a comment might be kept

---

the linter allows you to either [ignore lines](https://github.com/editorconfig-checker/editorconfig-checker?tab=readme-ov-file#excluding-lines), as I did, or [ignore blocks](https://github.com/editorconfig-checker/editorconfig-checker?tab=readme-ov-file#excluding-blocks), but then it's 2 comments - before and after the block

---

Yes, that's why I consider adding the code needed for editorcheck-disable-next-line

---

Let's leave the current PR as it is for now? And fix it later, when support for editorcheck-disable-next-line is added?

---

As discussed directly on Slack

You could enable

https://revive.run/r#line-length-limit

So you can configure it via golangci-lint 

https://golangci-lint.run/usage/linters/#revive

```yaml
 linters-settings:
     revive:
     rules:
       - name: line-length-limit
         arguments: [120]
```

So I would say add this linter in your PR, disable the line length check for go files in .editorconfig

And you will be fine